specification/columns/pricingcategory.md

```diff
@@ -8,7 +8,7 @@ The PricingCategory column adheres to the following requirements:
 * PricingCategory MUST NOT be null when [ChargeClass](#chargeclass) is not "Correction" and [ChargeCategory](#chargecategory) is "Usage" or "Purchase", MUST be null when ChargeCategory is "Tax", and MAY be null for all other combinations of ChargeClass and ChargeCategory.
 * PricingCategory MUST be one of the allowed values.
 * PricingCategory MUST be "Standard" when pricing is predetermined at the agreed upon rate for the [billing account](#glossary:billing-account).
-* PricingCategory MUST be "Committed" when [CommitmentDiscountId](#commitmentdiscountid) is not null.
+* PricingCategory MUST be "Committed" when [CommitmentDiscountId](#commitmentdiscountid) is not null, except when ChargeCategory is "Purchase" and the [ResourceId](#resourceid) matches the CommitmentDiscountId.
```

Comment:
Option D?
```suggestion
* PricingCategory MUST be "Committed" when [CommitmentDiscountId](#commitmentdiscountid) is not null and ChargeCategory is "Usage".
```

Reply:
Are we absolutely certain that Commitment discounts, including any future ones, cannot be applied to purchase-related records?

Reply:
Yes, 💯. A commitment discount's reduction only applies to usage rows. Any additional reduction to the purchase is a negotiated discount.

Reply:
I'm not sure that's a fair assumption. That's just how it works today for the major cloud providers. We don't know if anyone's using commitment discounts for purchases. I don't think we should limit commitment discounts to usage only. As an example, a domain registrar may offer commitment discounts for organizations who purchase 1,000 domains, which would be a bulk of purchases and not usage. These are valid discounts for committing to a specific quantity of purchases. We should support that.

Reply:
I'm fine with the proposed solution as is, so we can resolve this comment.